packaging/RPMS/Fedora/rabbitmq-server.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
Shouldn't the epmd dependency be specified here using `Requires` (or at least `Wants`)?